### PR TITLE
#475: - displaying the error on the modals views as create address

### PR DESCRIPTION
**Pull request by hani-ibrahim**
working on #475

### CODE REVIEW

**Comment by hani-ibrahim** [comment]
👍

File: AtlasUI/AtlasUI/Utils/UserMessage.swift [hunk 1]
```diff
@@ -41,41 +41,50 @@ struct UserMessage {
...
-    private static func displayBanner(error: UserPresentable, on viewController: UIViewController) {
+    private static func displayBanner(error: UserPresentable) {
+        let viewController = errorPresenterViewController
         bannerErrorViewController.removeFromParentViewController()
         bannerErrorViewController.view.removeFromSuperview()
 
-        viewController.addChildViewController(bannerErrorViewController)
-        viewController.view.addSubview(bannerErrorViewController.view)
+        viewController?.addChildViewController(bannerErrorViewController)
+        viewController?.view.addSubview(bannerErrorViewController.view)
```
Comment (bonkey) [inline AtlasUI/AtlasUI/Utils/UserMessage.swift]: would be clearer with `if let viewController = errorPresenterViewController {}`

File: AtlasUI/AtlasUI/Utils/UserMessage.swift [hunk 2]
```diff
@@ -41,41 +41,50 @@ struct UserMessage {
...
     }
 
-    private static func displayFullScreen(error: UserPresentable, on viewController: UIViewController) {
+    private static func displayFullScreen(error: UserPresentable) {
+        let viewController = errorPresenterViewController
         let navigationController = UINavigationController(rootViewController: fullScreenErrorViewController)
-        viewController.addChildViewController(navigationController)
-        viewController.view.addSubview(navigationController.view)
+        viewController?.addChildViewController(navigationController)
+        viewController?.view.addSubview(navigationController.view)
```
Comment (bonkey) [inline AtlasUI/AtlasUI/Utils/UserMessage.swift]: would be clearer with `if let viewController = errorPresenterViewController {}`

**Comment by bonkey** [comment]
👍

File: AtlasUI/AtlasUI/Utils/UserMessage.swift (hunk 1 above)
Comment (bonkey) [inline AtlasUI/AtlasUI/Utils/UserMessage.swift]: 👍

**Pull request merged**